app/src/androidTest/java/org/kiwix/kiwixmobile/search/SearchFragmentTest.kt

```diff
@@ -41,18 +44,25 @@ import org.kiwix.kiwixmobile.testutils.TestUtils.isSystemUINotRespondingDialogVi
 import java.io.File
 import java.io.FileOutputStream
 import java.io.OutputStream
+import java.net.URI
 
 class SearchFragmentTest : BaseActivityTest() {
 
+  private val stackExchangeZimFileUrl =
+    "https://download.kiwix.org/zim/stack_exchange/academia.stackexchange.com_en_all_2023-05.zim"
```

Comment:
@MohitMaliDeveloper Will this  Url always be valid in future?

Reply:
@gouri-panda  if this zim will update with other names in the future then this URL will become invalid.

Reply:
@kelson42 Can you please provide me the zim file link so I can update that in our test case?